src/containers/App.tsx

```diff
@@ -57,21 +58,28 @@ const App: FC = () => {
       <Layout>
         <Switch>
           <Route exact path="/" component={withSuspense(DeveloperPortal)} />
-          <Route exact path="/whitepaper" component={LivingWhitepaper} />
-          <Route exact path="/whitepaper/principal-entities/:chapter?" component={PrincipalEntities} />
-          <Route exact path="/whitepaper/principal-events/:chapter?" component={PrincipalEventsAndProcesses} />
-          <Route exact path="/whitepaper/architecture/:chapter?" component={ArchitectureDeepDive} />
-          <Route exact path="/projects" component={Projects} />
-          <Route exact path="/projects/rules" component={ProjectRulesAndGuidelines} />
-          <Route exact path="/projects/approved-projects/:projectId?" component={ApprovedProjects} />
-          <Route path="/api/bank-api/:chapter?" component={BankApi} />
-          <Route path="/api/confirmation-validator-api/:chapter?" component={ConfirmationValidatorApi} />
-          <Route path="/api/primary-validator-api/:chapter?" component={PrimaryValidatorApi} />
-          <Redirect path="/api" to="/api/bank-api" />
-          <Route exact path="/sdks-and-libraries" component={SdkAndLibraries} />
-          <Route path="/guidelines" component={Guidelines} />
-          <Route path="/terms-of-use" component={TermsOfUse} />
-          <Route path="/privacy-policy" component={PrivacyPolicy} />
+          <Route exact path={ROUTES.whitepaper.home} component={LivingWhitepaper} />
+          <Route exact path={`${ROUTES.whitepaper.principalEntities}/:chapter?`} component={PrincipalEntities} />
+          <Route
+            exact
+            path={`${ROUTES.whitepaper.principalEvents}/:chapter?`}
+            component={PrincipalEventsAndProcesses}
+          />
+          <Route exact path={`${ROUTES.whitepaper.architecture}/:chapter?`} component={ArchitectureDeepDive} />
+          <Route exact path={ROUTES.projects.home} component={Projects} />
+          <Route exact path={ROUTES.projects.rules} component={ProjectRulesAndGuidelines} />
+          <Route exact path={`${ROUTES.projects.approvedProjects}/:projectId?`} component={ApprovedProjects} />
+          <Route path={`${ROUTES.tools.apis}/bank-api/:chapter?`} component={BankApi} />
+          <Route
+            path={`${ROUTES.tools.apis}/confirmation-validator-api/:chapter?`}
+            component={ConfirmationValidatorApi}
+          />
+          <Route path={`${ROUTES.tools.apis}/primary-validator-api/:chapter?`} component={PrimaryValidatorApi} />
+          <Redirect path={ROUTES.tools.apis} to={`${ROUTES.tools.apis}/api/bank-api`} />
```

Comment:
should be `to={`${ROUTES.tools.apis}/bank-api`}`

Reply:
ah sorry for the mistakes, it's around 2+ am here and my brain isn't functioning that well

Reply:
yeah I guessed it xD